content/200-concepts/100-components/01-prisma-schema/06-relations/07-referential-actions.mdx

```diff
@@ -8,6 +8,401 @@ preview: true
+
+Prisma supports the following referential actions:
+
+- [`Cascade`](#cascade)
+- [`Restrict`](#restrict)
+- [`NoAction`](#noaction)
+- [`SetNull`](#setnull)
+- [`SetDefault`](#setdefault)
+
+### Referential action defaults
+
+If you do not specify a referential action, Prisma uses the following defaults:
+
+| Clause     | Optional relations | Mandatory relations |
+| :--------- | :----------------- | :------------------ |
+| `onDelete` | `SetNull`          | `Restrict`          |
+| `onUpdate` | `Cascade`          | `Cascade`           |
+
+For example - in the following schema, all `Post` records must be connected to an `author`. The schema **does not explicitly define referential actions** on the `author` field, which means that default referential actions apply (see SQL tab):
+
+<TabbedContent tabs={[<FileWithIcon text="schema.prisma" icon="prisma"/>, <FileWithIcon text="SQL" icon="database"/>]}>
+<tab>
+
+```prisma highlight=4;normal
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  author    User    @relation(fields: [authorId], references: [id])
+  authorId  Int
+}
+
+model User {
+  id    Int         @id @default(autoincrement())
+  posts Post[]
+}
+```
+
+</tab>
+<tab>
+
+```sql highlight=5;normal
+CREATE TABLE "Post" (
+  "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  "title" TEXT NOT NULL,
+  "authorId" INTEGER NOT NULL,
+  FOREIGN KEY ("authorId") REFERENCES "User" ("id") ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
+CREATE TABLE "User" (
+  "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+);
+```
+
+</tab>
+</TabbedContent>
+
+## Caveats
+
+The following caveats apply:
+
+*  Referential actions are **not currently** supported on [implicit many-to-many relations](./#implicit-vs-explicit-many-to-many-relations). To use referential actions, you must define an explicit many-to-many relation and define your referential actions on the [join table](./#how-to-use-a-relation-table-with-a-many-to-many-relationship).
+
+## Types of referential actions
+
+The following table shows which referential action each database supports. 
+
+<Admonition type="info">
+
+Note that [`NoAction`](#noaction) can be used in replace of [`Restrict`](#restrict) when working with an SQLServer database.
+
+</Admonition>
+
+| Database   | Cascade | Restrict | NoAction |  SetNull | SetDefault  |
+| :--------- | :------ | :------- | :------- | :------- | :---------- |
+| PostgreSQL | **✔️**  | **✔️**  | **✔️**  | **✔️**   | **✔️**     |
+| MySQL      | **✔️**  | **✔️**  | **✔️**  | **✔️**   | **✔️**     |
+| SQLite     | **✔️**  | **✔️**  | **✔️**  | **✔️**   | **✔️**     |
+| SQLServer  | **✔️**  | **❌**  | **✔️**  | **✔️**   | **✔️**     |
+
+### Cascade
+
+- `onDelete: Cascade` Deleting a referenced record will trigger the deletion of referencing record.
+
+- `onUpdate: Cascade` Updates the relation scalar field if the referenced scalar fields of the dependant record are updated.
```

Comment:
```suggestion
- `onUpdate: Cascade` Updates the relation scalar fields if the referenced scalar fields of the dependant record are updated.
```

Just to match pluralization. An alternative we could consider is field(s) as the most common case is for ids to be single fields.